isbn/lib.php

```diff
@@ -69,7 +69,8 @@
     'produktSigel' => '//feld[@nr="078" and @ind="e"]/uf[@code="a"]',
 );
 
-function performMapping($map, $outputXml) {
+function performMapping($map, $outputXml)
+{
```

Comment:
Yes, PSR-2 puts them on a new line for functions. See http://www.php-fig.org/psr/psr-2/, chapter 4.3.